app/models/user.rb

```diff
@@ -8,9 +8,10 @@ class User < ApplicationRecord
   validates :first_name, presence: true, on: :update_account
   validates :last_name, presence: true, on: :update_account
   validates :organization_name, presence: true, on: :update_account
+  validates :email, presence: true, uniqueness: { case_sensitive: false }
```

Comment:
We should continue to validate the email length & format, which authlogic used to do for us. Please add those validations with corresponding specs. See https://github.com/binarylogic/authlogic/blob/master/doc/use_normal_rails_validation.md

Reply:
Added validation and test therof.